package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "@gnosis.pm/ui",
+  "version": "0.0.1",
+  "description": "Reusable React components and setup functions built with Typescript and love. From Gnosis.",
+  "main": "src/index.tsx",
+  "directories": {
+    "test": "tests"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/gnosis/ui.git"
+  },
+  "keywords": [
+    "gnosis",
+    "ui",
+    "ethereum",
+    "blockchain",
+    "typescript",
+    "javascript",
+    "react"
+  ],
+  "author": "W3stside <doohhello@gmail.com>, Velenir<>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/gnosis/ui/issues"
+  },
+  "homepage": "https://github.com/gnosis/ui#readme",
+  "scripts": {
+    "postinstall": "cd ./examples/ts-frontend && npm i && cd ../../",
+    "start": "NODE_ENV=development webpack-dev-server --mode development --hot",
+    "start:example-ts": "cd ./examples/ts-frontend && npm start && cd ../../",
+    "build": "rimraf ./dist && NODE_ENV=production webpack --mode production",
+    "eslint": "eslint . --ext .ts,.tsx,.js",
+    "eslint-fix": "eslint --fix . --ext .ts,.tsx,.js",
+    "stats": "NODE_ENV=production webpack --mode production --env.stats --profile --json > stats.json && webpack-bundle-analyzer ./stats.json ./dist",
+    "test": "jest --detectOpenHandles"
+  },
+  "dependencies": {
+    "@hot-loader/react-dom": "^16.8.6",
+    "@walletconnect/web3-provider": "^1.0.0-beta.32",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-hot-loader": "^4.12.10",
+    "styled-components": "^4.3.2",
+    "web3": "^2.0.0-alpha.1",
```

Comment:
@Velenir thoughts here?

Reply:
In my experience 2.0.0 is more stable than 1.0. Certainly has more fixes included. Remember they were both renamed from beta only a month ago. 1.0 is more stable in the sense that truffle and co. are using it, that's all

On a semi-related note, I think we may be better off not dealing with web3 at all, only provider. Leave web3 (or etherjs or whatever) instantiation to anyone using the library. Everything from accountsChange to WC session management can be done with provider only

Reply:
Yes, right now is a copy of 1.0 but if they write that is not mainnet ready it's because they expect breaking changes.

I think the app we have to do is super basic, it should be ok to work with "not the last version". If there's a big reason to jump for the unstable, we could review it, otherwise I think is not worth it at this point. We can revisit this in a few months.